library/globals.inc.php

```diff
@@ -3092,5 +3092,40 @@ function gblTimeZones()
             xl('Select to print patient address labels from popups in left nav screen.')
         ),
 
+        'env_x_width' => array(
+            xl('Envelope Width in mm'),
+            'num',                           // data type
+            '104.775',                              // default = false
+            xl('In Portrait mode, determines the width of the envelope along the x-axis in mm')
+        ),
+
+        'env_y_height' => array(
+            xl('Envelope Height in mm'),
+            'num',                           // data type
+            '241.3',                              // default = false
+            xl('In Portrait mode, determines the height of the envelope along the y-axis in mm')
+        ),
+
+        'env_font_size' => array(
+            xl('Font Size in Pt'),
+            'num',                           // data type
+            '14',                              // default = false
+            xl('Sets the font of the address text on the envelope in mm')
+        ),
+
+        'env_x_dist' => array(
+            xl('Envelope x-axis starting pt'),
+            'num',                           // data type
+            '65',                              // default = false
+            xl('Distance from the \'top\' of the envelope in mm')
+        ),
+
+        'env_y_dist' => array(
+            xl('Envelope y-axis starting pt'),
+            'num',                           // data type
+            '220',                              // default = false
+            xl(' Distance from the right most edge of the envelope in portrait position in mm')
+        ),
+
```

Comment:
General comment about use of global options:

We are facing operational issue with access control and this will be perfect example.  Typically the front office will need to tweak the envelope measurements - probably on a regular basis as the stock and/or printer changes.  But they won't have access.  So we are left with choice of making this user option or giving them permission to global setting.

May be we need page specific settings on the page itself that can be saved and accessed from user_settings.